velox/connectors/hive/storage_adapters/hdfs/tests/CMakeLists.txt

```diff
@@ -28,7 +27,38 @@ target_link_libraries(
   velox_exec
   GTest::gtest
   GTest::gtest_main
-  GTest::gmock)
+  GTest::gmock
+  CURL::libcurl)
 
 target_compile_options(velox_hdfs_file_test
                        PRIVATE -Wno-deprecated-declarations)
+
+add_executable(velox_hdfs_insert_test InsertIntoHdfsTest.cpp
+                                      HdfsMiniCluster.cpp HdfsUtilTest.cpp)
+
+add_test(velox_hdfs_insert_test velox_hdfs_insert_test)
+
+target_link_libraries(
+  velox_hdfs_insert_test
+  velox_file
+  velox_hdfs
+  velox_core
+  velox_exec_test_lib
+  velox_hive_connector
+  velox_dwio_common_exception
+  velox_exec
```

Comment:
I only see folly, hdfs, gtest and exec being used. Please only link used libs.

Reply:
@assignUser Good catch. I have removed the unused libs. Thanks.